vpr/src/route/route_tree_timing.cpp

```diff
@@ -252,7 +266,7 @@ void add_route_tree_to_rr_node_lookup(t_rt_node* node) {
 }
 
 static t_rt_node*
-add_subtree_to_route_tree(t_heap* hptr, t_rt_node** sink_rt_node_ptr) {
+add_subtree_to_route_tree(t_heap* hptr, int target_pin, t_rt_node** sink_rt_node_ptr) {
```

Comment:
Rename target_pin to target_net_pin_index, update comment to mention it, and move comment above function.

Reply:
Done.